packages/casbin/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@hono/casbin",
+  "version": "1.0.1",
```

Comment:
Please set the version number as `0.0.0`. The changeset will set it automatically.

Reply:
@yusukebe 
Thank you, I fixed it.